checks.d/docker_daemon.py

```diff
@@ -69,6 +69,13 @@
         },
     },
     {
+        "cgroup": "cpu",
+        "file": "cpu.stat",
+        "metrics": {
+            "nr_throttled": ("docker.cpu.throttled", GAUGE)
```

Comment:
I believe this is always increasing value, in which case a `rate` might be a better data type... not totally sure though.